android/app/src/main/java/app/organicmaps/downloader/ChunkTask.java

```diff
@@ -132,6 +135,11 @@ protected Integer doInBackground(Void... p)
       if (isCancelled())
         return CANCELLED;
 
+      // Fix missing root certificates for HTTPS connections on Android 7 and below:
+      // https://community.letsencrypt.org/t/letsencrypt-certificates-fails-on-android-phones-running-android-7-or-older/205686
+      if (android.os.Build.VERSION.SDK_INT <= android.os.Build.VERSION_CODES.N && mUrl.startsWith("https://"))
+        ((HttpsURLConnection) urlConnection).setSSLSocketFactory(MwmApplication.sslSocketFactory);
```

Comment:
Is possible sslSocketFactory  == null ok here?

Reply:
I would prefer to have a crash here on Android 7 and lower, so we can debug why certificates did not load properly.

Reply:
Put assert in getSslSocketFactory then?

Reply:
asserts do not work on Android.

setSSLSocketFactory will throw an illegal argument exception, so we will know when it fails in the play console.